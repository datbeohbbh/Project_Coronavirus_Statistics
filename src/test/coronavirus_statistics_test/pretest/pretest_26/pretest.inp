34 28
q 1 18 41
q 1 3 291
q 1 32 350
q 1 28 256
q 1 26 472
? 4 2
? 6
q 1 29 67
q 1 17 156
q 1 2 133
q 1 13 140
? 6
q 1 24 365
? 7
q 2 24 210
q 1 32 40
? 6
? 4 7
? 4 3
q 1 29 479
q 1 18 141
? 4 7
? 4 10
? 4 9
? 4 5
? 4 10
? 4 1
? 4 9