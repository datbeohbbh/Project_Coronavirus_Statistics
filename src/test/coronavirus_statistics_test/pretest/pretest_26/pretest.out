Query ? 4 2 :
The answer is the city with the index 28 
- Information of the city 28 -
-> Number of current patients : 256
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 7 :
Currently, no deaths have been recorded

Query ? 6 :
The city with the highest recovery is city with the index 24
Quantity: 210 people

Query ? 4 7 :
The answer is the city with the index 28 
- Information of the city 28 -
-> Number of current patients : 256
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 133
-> Has recovered : 0
-> Was dead : 0

Query ? 4 7 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 291
-> Has recovered : 0
-> Was dead : 0

Query ? 4 10 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 546
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 26 
- Information of the city 26 -
-> Number of current patients : 472
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 18 
- Information of the city 18 -
-> Number of current patients : 182
-> Has recovered : 0
-> Was dead : 0

Query ? 4 10 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 546
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 133
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 26 
- Information of the city 26 -
-> Number of current patients : 472
-> Has recovered : 0
-> Was dead : 0

